letsencrypt/auth_handler.py

```diff
@@ -113,6 +113,18 @@ def _solve_challenges(self):
                     cont_resp = self.cont_auth.perform(self.cont_c)
                 if self.dv_c:
                     dv_resp = self.dv_auth.perform(self.dv_c)
+                    for achall, response in zip(self.dv_c, dv_resp):
+                        if isinstance(response, challenges.HTTP01Response):
+                            if not response.simple_verify(
+                                achall.chall, achall.domain,
```

Comment:
Continuation lines in the `if` condition should be indented by additional 4 spaces. Same below (in total 4 lines have wrong indentation). Otherwise they are indented to the same level as `logger.warning` statement...